src/__tests__/canonicalFields.int.test.ts

```diff
@@ -1,20 +1,23 @@
 import request from 'supertest';
 import { app } from '../app';
-import {
-  prepareDatabaseForCurrentWorker,
-  cleanupDatabaseForCurrentWorker,
-} from '../test/harnessFunctions';
+import { db } from '../database';
 
 const agent = request.agent(app);
 
 describe('/canonicalFields', () => {
   describe('/', () => {
-    it('should return HTTP Status Code 200 OK', async () => {
-      await prepareDatabaseForCurrentWorker();
+    it('returns an empty array when no data is present', async () => {
       await agent
+        .get('/canonicalFields')
+        .expect(200, []);
+    });
+
+    it('returns all canonical fields present in the database', async () => {
+      await db.sql('fixtures.canonicalFields.returnsAllCanonicalFields.seed');
+      const result = await agent
         .get('/canonicalFields')
         .expect(200);
-      await cleanupDatabaseForCurrentWorker();
+      expect(result.body).toMatchSnapshot();
```

Comment:
On first glance, the snapshot doesn't look like literal JSON to me, but I am not sure if that's a downside yet.